dt-assets/parts/modals/modal-record-history.php

```diff
@@ -0,0 +1,56 @@
+<?php
+global $post;
+?>
+
+<div class="large reveal" id="record_history_modal" data-reveal data-reset-on-close style="padding-bottom: 50px;">
+    <h2><?php echo esc_html( sprintf( _x( '%s Record History', 'Record History', 'disciple_tools' ), $post->post_title ) ); ?></h2>
+    <hr>
+
+    <div class="grid-container">
+        <div class="grid-x">
+            <div class="cell small-4">
+                <table style="border: none;  max-width: 300px;">
+                    <tbody style="border: none;">
+                    <tr>
+                        <td colspan="2">
+                            <h4><?php esc_html_e( 'Filter To Date', 'disciple_tools' ) ?></h4>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td colspan="2">
+                            <select id="record_history_calendar">
+                                <option value="" selected disabled>--- <?php esc_html_e( 'select date to filter by', 'disciple_tools' ) ?> ---</option>
+                            </select>
+                        </td>
+                    </tr>
+                    <tr style="border: none;">
+                        <td style="vertical-align: top;">
+                            <span><?php echo esc_html( _x( 'Show All', 'Show All', 'disciple_tools' ) ) ?></span>
```

Comment:
No need to add context if the string is the same as the context.
Adding the context can result in an extra string to translate